packages/react-native/scripts/codegen/__tests__/generate-specs-cli-executor-test.js

```diff
@@ -82,7 +78,5 @@ describe('generateSpec', () => {
       packageName,
       libraryType,
     );
-
-    expect(mkdirpSyncInvoked).toBe(1);
```

Comment:
Looks like this test file was partially updated in https://github.com/facebook/react-native/pull/41956

If only single invocation of mkdir Sync is needed as per changes in that PR, the verification of second and third invocation of mkdirSync call is not needed in mock function above.

Ideally, this should be fixed in a separate PR as a miss, so that it's not part of mkdirp removal PR - but the existing maintainers will likely comment on what would be the right path to follow.

Reply:
Irrespective of how it's fixed, the update should test that fs.mkdir function was called once in this replacement.
This can be done using the mock recommended in https://github.com/facebook/react-native/pull/46388#discussion_r1749544068

Reply:
Great point! I've simplified the test to verify that mkdirSync is called only once, as the function logic only creates the outputDirectory. I removed the extra checks for multiple calls to align with the actual behaviour. 
Let me know if further adjustments are needed!

Reply:
Verified update in commit [`b8c406c` (#46388)](https://github.com/facebook/react-native/pull/46388/commits/b8c406cc46257db86c13a7e9f1ebc650b98a2a37).

I don't have permission to resolve this thread though.